locales-zh-TW.xml

```diff
@@ -148,6 +148,7 @@
 
     <!-- VERB ROLE FORMS -->
     <term name="author" form="verb">著</term>
```

Comment:
According to today's straw poll, the term is not applied to author names in citations in TW, nor in CN.